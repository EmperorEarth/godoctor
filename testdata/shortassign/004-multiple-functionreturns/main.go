// <<<<< shortassign,11,3,11,13,fail
package main

import "fmt"

func f() (int, float64) {
	return 1, 2.3
}

func main() {
  i, x := f()
  fmt.Println(i, x)
}
