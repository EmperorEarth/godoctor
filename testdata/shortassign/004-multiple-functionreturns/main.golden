// <<<<< shortassign,11,3,10,13,fail
package main

import "fmt"

func f() (int, float64) {
	return 1, 2.3
}

func main() {
  var i, x  = f()
  fmt.Println(i, x)
}
